src/unreal_plugin/Source/UnrealDeadlineCloudService/Private/DeadlineCloudJobSettings/DeadlineCloudDetailsWidgetsHelper.cpp

```diff
@@ -0,0 +1,318 @@
+            ];
+    }
+
+private:
+    void HandleTextChanged(const FText& NewText)
+    {
+        PathProperty->SetValue(NewText.ToString());
+    }
+
+    FText GetText() const
+    {
+        FString String;
+        PathProperty->GetValue(String);
+
+        return FText::FromString(String);
+    }
+
+    TSharedPtr<IPropertyHandle> PathProperty;
+};
+
+class SDeadlineCloudIntWidget : public SCompoundWidget
+{
+public:
+    SLATE_BEGIN_ARGS(SDeadlineCloudIntWidget) {}
+        SLATE_ARGUMENT(TSharedPtr<IPropertyHandle>, PropertyHandle)
+    SLATE_END_ARGS()
+
+    void Construct(const FArguments& InArgs)
+    {
+        Property = InArgs._PropertyHandle;
+
+        ChildSlot
+            [
+                SNew(SHorizontalBox)
+                    + SHorizontalBox::Slot()
+                    .FillWidth(1.0f)
+                    .VAlign(VAlign_Center)
+                    [
+                        SNew(SNumericEntryBox<int32>)
+                            .Font(IDetailLayoutBuilder::GetDetailFont())
+                            .AllowSpin(false)
+                            .MinDesiredValueWidth(50.0f)
+                            .Value_Lambda([this]
+                                {
+                                    FString String;
+                                    Property->GetValue(String);
+                                    return FCString::Atoi(*String);
+                                })
+                            .OnValueCommitted_Lambda([this](int32 Value, ETextCommit::Type InCommitType)
+                                {
+                                    Property->SetValue(FString::FromInt(Value));
+                                })
+                    ]
+            ];
+    }
+
+private:
+
+    TSharedPtr<IPropertyHandle> Property;
+};
+
+class SDeadlineCloudFloatWidget : public SCompoundWidget
+{
+public:
+    SLATE_BEGIN_ARGS(SDeadlineCloudFloatWidget) {}
+        SLATE_ARGUMENT(TSharedPtr<IPropertyHandle>, PropertyHandle)
+    SLATE_END_ARGS()
+
+    void Construct(const FArguments& InArgs)
+    {
+        Property = InArgs._PropertyHandle;
+
+        ChildSlot
+            [
+                SNew(SHorizontalBox)
+                    + SHorizontalBox::Slot()
+                    .FillWidth(1.0f)
+                    .VAlign(VAlign_Center)
+                    [
+                        SNew(SNumericEntryBox<float>)
+                            .Font(IDetailLayoutBuilder::GetDetailFont())
+                            .AllowSpin(false)
+                            .MinDesiredValueWidth(50.0f)
+                            .Value_Lambda([this]
+                                {
+                                    FString String;
+                                    Property->GetValue(String);
+
+                                    return FCString::Atof(*String);
+                                })
+                            .OnValueCommitted_Lambda([this](float Value, ETextCommit::Type InCommitType)
```

Comment:
nit: `InCommitType` variable doesn't seem to be used. Maybe we could omit it?
```suggestion
                            .OnValueCommitted_Lambda([this](float Value, ETextCommit::Type)
```

Reply:
will fix